lib/service/businessLogic.js

```diff
@@ -18,6 +18,10 @@ const notImplementedHandler = require('./notImplementedHandler');
 
 const registeredFunctions = new Map();
 
+function getHandlers() {
+  return [...registeredFunctions.keys()];
```

Comment:
@galniv `registeredFunctions.keys()` is an es6 map function call which gets all of the keys in a `map` object. `...` is the es6 spread operator which essentially allows me to initialize a new array containing the `map.keys()` literals